src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -25,12 +26,14 @@ struct AllSkyRadiation <: AbstractRRTMGPMode
     idealized_clouds::Bool
     add_isothermal_boundary_layer::Bool
     aerosol_radiation::Bool
+    reset_rng_seed::Bool
```

Comment:
Can you please add a docstring/comment to this? Something like

```suggestion
    "reset_rng_seed is a boolean that controls whether the seed for the random number generator is reset every time the radiation callback is called. This is accomplished by ABC and it is useful for XYZ. Disable this for production runs."
    reset_rng_seed::Bool
```

Same below